files/teaching-demo-rubric.md

```diff
@@ -3,28 +3,28 @@ This rubric is provided as a guide for Trainers evaluating potential new instruc
 Trainer. As such, deviation from this rubric is encouraged as needed to accurately assess the trainee's preparation and instructional
 skills.
 
-In general, trainees who have three or more marks in the "Negative" columns below should be asked to redo their demonstration. Even one 
-mark in a "Negative" column can be justification for asking a trainee to redo their demonstration if the problem is significant. As always,
+In general, these demonstrations have a very low (<5%) failure rate. When a trainee is asked to redo a demo, it is simply a matter of correction -- an error to embrace and to learn from -- not rejection. Trainees who have three or more marks in the "Negative" columns below should probably be asked to redo their demonstration. Even one 
+mark in a "Negative" column can be considered justification for asking a trainee to redo their demonstration if the problem is significant. As always,
 Trainers should use their own judgement when applying this rubric in individual cases.  
 
 |Positive Content|Negative Content|
 |------|---------------------|
-Trainee uses the official curricular materials as written or with minor deviations|Trainee deviates significantly from the official curricular materials|
-Trainee places content in context and explains why it is relevant/useful to the learner|Trainee jumps into the content without explaining context|
-Trainee obviously knows the material they are teaching|Trainee makes factual errors in teaching the content and doesn't correct themselves|
+Uses official curriculum with only minor deviations|Deviates significantly or doesn't use official curriculum|
```

Comment:
- The word "official" seems pretty formal - "standard", maybe?

Reply:
+1